### PR TITLE
Add minification and gzipping

### PR DESCRIPTION
**Pull request by michaelrhodes**
I feel like some of this might be contentious, but may as well get the ball rolling on addressing [issue #13](https://github.com/jfhbrook/wzrd.in/issues/13). Note that I’ve tried to be really defensive so that on the off chance minification fails, the bundle can still be served.

Hope this helps! :)

### CODE REVIEW

File: bundler/minify.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+var minify = require('uglify-js').minify;
+
+//
+// Run uglify-js
+//
+module.exports = function (env, bundle, cb) {
+  env.log.info('minify: running through uglify-js.');
+
+  try {
+    var result = minify(bundle, { fromString: true });
```
Comment (jfhbrook) [inline bundler/minify.js]: It looks like minify is synchronous, so the exported wrapper should probably be sync too?

**Comment by jfhbrook** [comment]
Oh neat! Just the one comment.

File: bundler/minify.js (hunk 1 above)
Comment (michaelrhodes) [inline bundler/minify.js]: Oh, good call.

File: bundler/index.js [hunk 2]
```diff
@@ -120,7 +121,9 @@ module.exports = function bundler(opts) {
 
               browserify(env, pkg, function (err, bundle) {
                 if (err) return handleError(err);
-                return finish(err, bundle, json);
+                if (pkg.debug) return finish(err, bundle, json);
+
+                finish(err, minify(env, bundle) || bundle, json);
```
Comment (jfhbrook) [inline bundler/index.js]: You gotta catch any potential minify errors, something like:

```js
var minified = null;
try {
  minified = minify(env, bundle);
} catch (err) {
  env.log.error(err); // I forget the exact API of the logger, unfortunately
  return finish(err, bundle, json);
}
finish(err, minified || bundle, json);
```

Comment (michaelrhodes) [inline bundler/index.js]: I’m actually doing that in `bundler/minify.js`

Comment (jfhbrook) [inline bundler/index.js]: Hmm. I suppose you're right. Yeah, that should be fine.

**Comment by jfhbrook** [comment]
Thanks!

**Comment by michaelrhodes** [comment]
Awesome, no worries!

**Pull request merged**